transform.js

```diff
@@ -92,27 +98,59 @@ function transform (filename, options) {
     }
 
     function extractTemplateNodes (node) {
-      if (node.type !== 'TemplateLiteral') return
-      if (!node.parent || !node.parent.tag) return
-      if (node.parent.tag.name !== mname) return
+      if (node.type === 'VariableDeclarator' && node.init && isBabelTemplateDefinition(node.init)) {
+        // Babel generates helper calls like
+        //    _taggedTemplateLiteral([":host .class { color: hotpink; }"], [":host .class { color: hotpink; }"])
+        // we only keep the "cooked" part
+        babelTemplateObjects[node.id.name] = node.init.arguments[0]
+      }
 
-      const css = [ node.quasis.map(cooked) ]
-        .concat(node.expressions.map(expr)).join('').trim()
+      if (node.type === 'TemplateLiteral' && node.parent && node.parent.tag) {
+        if (node.parent.tag.name !== mname) return
 
-      const val = {
-        css: css,
-        filename: filename,
-        opts: xtend(opts),
-        node: node.parent
+        const css = [ node.quasis.map(cooked) ]
+          .concat(node.expressions.map(expr)).join('').trim()
+
+        const val = {
+          css: css,
+          filename: filename,
+          opts: xtend(opts),
+          node: node.parent
+        }
+
+        nodes.push(val)
       }
 
-      nodes.push(val)
+      if (node.type === 'CallExpression' && node.callee.type === 'Identifier' && node.callee.name === mname) {
+        let elements
+        if (node.arguments[0] && node.arguments[0].type === 'ArrayExpression') {
+          // Buble generates code like
+          //     sheetify([":host .class { color: hotpink; }"])
+          elements = node.arguments[0].elements
+        } else if (node.arguments[0] && node.arguments[0].type === 'Identifier') {
+          // Babel generates code like
+          //    sheetify(_templateObject)
+          elements = babelTemplateObjects[node.arguments[0].name].elements
+        }
+
+        if (elements) {
+          const val = {
+            css: elements.map(function (part) { return part.value }),
```

Comment:
this means the `val.css` property is an array, but i think it should be a string?

Reply:
True, it probably somehow gets auto-joined when it shouldn't....